frontend/components/TableContainer/DataTable/TextCell/TextCell.tsx

```diff
@@ -1,26 +1,30 @@
+import classnames from "classnames";
 import { uniqueId } from "lodash";
 import React from "react";
 import ReactTooltip from "react-tooltip";
 import { COLORS } from "styles/var/colors";
 import { DEFAULT_EMPTY_CELL_VALUE } from "utilities/constants";
 
+const baseClass = "text-cell";
 interface ITextCellProps {
   value?: string | number | boolean | { timeString: string } | null;
   formatter?: (val: any) => React.ReactNode; // string, number, or null
   /** adds a greyed styling to the cell. This will italicise and add a grey
    * color to the cell text.
    * @default false
    */
-  greyed?: boolean;
-  classes?: string;
+  greyAndItalic?: boolean;
```

Comment:
I think this will be better represented with two props, `greyed` and `italic. its an easy change that allows us to style something as only greyed in the future if needed